code/modules/customitems/item_defines.dm

```diff
@@ -2089,4 +2089,15 @@ All custom items with worn sprites must follow the contained sprite system: http
 	icon_override = 'icons/obj/custom_items/freedom_coat.dmi'
 	icon_state = "freedom_coat"
 	item_state = "freedom_coat"
+	contained_sprite = TRUE
+
+/obj/item/clothing/head/fluff/schlosser_hat 
```

Comment:
This needs a comment with the player, aka 

```suggestion
/obj/item/clothing/head/fluff/schlosser_hat // Schlosser - NewOriginalSchwann
```

Just like every other item in this file